misaligned.c

```diff
@@ -16,6 +16,7 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    assert(result == 5);

```

Comment:
Why result should be 5 immediately after asserting result == 25?
we need to assert here what is expected from production code. 
The tests should include how the color codes are printed and their alignment also. Please consider the pair with single and double digits and its effect on alignment.